scp/queues.php

```diff
@@ -72,11 +73,13 @@
                 if ($queue->save()) $updated++;
                 break;
             case 'delete':
-                if ($queue->delete()) $updated++;
+                if ($queue->getId() == $cfg->get('default_ticket_queue'))
```

Comment:
Please use getter `$cfg->getDefaultTicketQueueId()`. Also you don't need to make `$cfg` global.